crates/autopilot/src/boundary/mod.rs

```diff
@@ -28,17 +28,12 @@ pub mod order;
 
 /// Builds a web3 client that bufferes requests and sends them in a
 /// batch call.
-pub fn buffered_web3_client(ethrpc: &Url) -> Web3 {
-    let ethrpc_args = shared::ethrpc::Arguments {
-        ethrpc_max_batch_size: 20,
-        ethrpc_max_concurrent_requests: 10,
-        ethrpc_batch_delay: Default::default(),
-    };
+pub fn buffered_web3_client(ethrpc: &Url, ethrpc_args: &shared::ethrpc::Arguments) -> Web3 {
```

Comment:
nit: `ethrpc_args` can be configured such that no buffering will happen so this function name is a bit misleading.